src/woo_publications/api/models/category.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from ordered_model.models import OrderedModel
+
+from ..constants import InformatieCategorieChoices
+
+
+class InformatieCategorie(OrderedModel):
+    identifier = models.URLField(_("Identifier"), max_length=255, unique=True)
```

Comment:
please make sure that all `verbose_name`'s start with a lower case letter - Django takes care of capitalizing that at the form layer

```suggestion
    identifier = models.URLField(_("identifier"), max_length=255, unique=True)
```